### PR TITLE
Fix decimal places issue in dmp.imitate and dmp.trajectory

### LINKED ISSUE #76: Error in DMP.imitate if dt has many decimal places
DMP.imitate throws error, if dt has many decimal places, e.g. 1.0/60 (60 Hertz).

### PR DESCRIPTION
**Pull request by ChristophPetzoldt**
Fixes #76 .

### CODE REVIEW

File: bolero/representation/dmp_behavior.py [hunk 1]
```diff
@@ -346,10 +346,12 @@ def imitate(self, X, Xd=None, Xdd=None, alpha=0.0,
             warnings.warn("Xdd is deprecated")
 
         X = X[:, :, 0].T.copy()
-        dmp.imitate(np.arange(0, self.execution_time + self.dt, self.dt),
-                    X, self.weights, self.widths, self.centers,
-                    alpha, self.alpha_y, self.beta_y, self.alpha_z,
-                    allow_final_velocity)
+        dmp.imitate(
+            np.linspace(0, self.execution_time + self.dt,
```
Comment (AlexanderFabisch) [inline bolero/representation/dmp_behavior.py]: is `self.execution_time + self.dt` what you want here?

**Comment by ChristophPetzoldt** [comment]
Just noticed that some unit tests fail after these changes. 
Assert_array_almost_equal is only True with decimal=2. Maybe, there is a problem with the linspace spreading...

File: bolero/representation/dmp_behavior.py (hunk 1 above)
Comment (ChristophPetzoldt) [reply bolero/representation/dmp_behavior.py]: You're right!

**Comment by ChristophPetzoldt** [comment]
Changing the linspace to:
`np.linspace(0, self.execution_time, round(self.execution_time + self.dt) / self.dt))`
solves the failures in the other unit tests.

**Comment by AlexanderFabisch** [comment]
rebased and merged with 88c3dfea384324489c0ec87048dfdc9e410430fb

**Pull request closed without merging**